pinot-core/src/main/java/org/apache/pinot/core/accounting/PerQueryCPUMemAccountantFactory.java

```diff
@@ -491,6 +504,16 @@ public Map<String, AggregatedStats> aggregate(boolean isTriggered) {
     public void postAggregation(Map<String, AggregatedStats> aggregatedUsagePerActiveQuery) {
     }
 
+    public boolean cancelQuery(String queryId) {
+      if (_queryCancelCallbacks.containsKey(queryId)) {
+        QueryCancelCallback callback = _queryCancelCallbacks.get(queryId);
+        callback.cancelQuery();
+        return true;
```

Comment:
Could we remove the entry here? or we expect to try to cancel them again in future?